drivers/display/display_sdl.h

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2018 dXplore
```

Comment:
2 comments:
* for consistency you may want to lower the copyright message one line
* is "dXplore" an actual legal entity?

Reply:
on the second point: no not yet